config/install/field.storage.node.field_length.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_length
+field_name: field_length
+entity_type: node
+type: integer
```

Comment:
This should be a "Number (decimal)" field type.
![vehicles_field_length_machine_name](https://github.com/notS0Solo/swapimigration/assets/1434979/ac10782f-46f9-4c29-9c7b-32b0dd89073c)

Reply:
Fixed